miso-web/src/main/webapp/pages/editPool.jsp

```diff
@@ -209,6 +209,13 @@
   jQuery(document).ready(function () {
     // Attach Parsley form validator
     Validate.attachParsley('#pool-form');
+
+    // store duplicate indices sequences
+    Pool.duplicateIndices = ${pool.hasDuplicateIndices()};
+    Pool.duplicateIndicesSequences = [];
```

Comment:
You can also just use `ObjectMapper` to make this into a JSON array. Then you can do a length check here to determine non-emptiness.